### ISSUE TITLE
Add link to the PSAT wiki on the CLI help message

### ISSUE THREAD

**Issue by fellyph** [GoogleChromeLabs/ps-analysis-tool#743]
## Feature Request

**Description:**
On the help message, list all the options around the PSAT CLI. To complement that information, it will be good to add a link to the PSAT Wiki documentation.

**now**

```
> ps-analysis-tool@0.9.0 cli
> node packages/cli/dist/main.js --help

Usage: main [options] [website-url]

CLI to test a URL for 3p cookies.

Arguments:
  website-url                 The URL of a single site to analyze

Options:
  -V, --version               output the version number
  -u, --url <url>             The URL of a single site to analyze
  -s, --source-url <url>       The URL of a sitemap or CSV to analyze
  -f, --file <path>           The path to a local file (CSV or XML sitemap) to analyze
  -n, --number-of-urls <num>  Limit the number of URLs to analyze (from sitemap or CSV)
  -d, --display               Flag for running CLI in non-headless mode
  -v, --verbose               Enables verbose logging
  -t, --tech                  Enables technology analysis
  -o, --out-dir <path>        Directory to store analysis data (JSON, CSV, HTML) without launching the dashboard
  -i, --ignore-gdpr           Ignore automatically accepting the GDPR banner if present
  -q, --quiet                 Skips all prompts; uses default options
  -c, --concurrency <num>     Number of tabs to open in parallel during sitemap or CSV analysis (default: 3)
  -w, --wait <num>            Number of seconds to wait after the page is loaded before generating the report
                              (default: 20000)
  -l, --locale <language>     Locale to use for the CLI, supported: en, hi, es, ja, ko, pt-BR (default: "en")
  -h, --help                  display help for command
``` 

**Adding a link to the bottom:**

```> ps-analysis-tool@0.9.0 cli
> node packages/cli/dist/main.js --help

Usage: main [options] [website-url]

CLI to test a URL for 3p cookies.

Arguments:
  website-url                 The URL of a single site to analyze

Options:
  -V, --version               output the version number
  -u, --url <url>             The URL of a single site to analyze
  -s, --source-url <url>       The URL of a sitemap or CSV to analyze
  -f, --file <path>           The path to a local file (CSV or XML sitemap) to analyze
  -n, --number-of-urls <num>  Limit the number of URLs to analyze (from sitemap or CSV)
  -d, --display               Flag for running CLI in non-headless mode
  -v, --verbose               Enables verbose logging
  -t, --tech                  Enables technology analysis
  -o, --out-dir <path>        Directory to store analysis data (JSON, CSV, HTML) without launching the dashboard
  -i, --ignore-gdpr           Ignore automatically accepting the GDPR banner if present
  -q, --quiet                 Skips all prompts; uses default options
  -c, --concurrency <num>     Number of tabs to open in parallel during sitemap or CSV analysis (default: 3)
  -w, --wait <num>            Number of seconds to wait after the page is loaded before generating the report
                              (default: 20000)
  -l, --locale <language>     Locale to use for the CLI, supported: en, hi, es, ja, ko, pt-BR (default: "en")
  -h, --help                  display help for command

To learn more, visit our wiki: https://github.com/GoogleChromeLabs/ps-analysis-tool/wiki.
```

**Motivation:**
This URL can guide users in reading more about the project and finding relevant information about the audit process. 

**User Story:**
As a user, I want to learn more about PSAT to take advantage of all PSAT features.

**Acceptance Criteria:**
- [x] add a link to the final of the help message

**Additional Information:**
[Include any additional information or context that may be helpful, such as related resources or examples.]

**Comment by fellyph** [comment]
This item was included in version 0.10:

<img width="1162" alt="Screenshot 2024-08-02 at 09 23 27" src="https://github.com/user-attachments/assets/dccf68fb-f6b3-40f6-945a-f6f7deacf909">

This feature request can be closed.

**Comment by maitreyie-chavan** [comment]
Addressed and released as a part of [v0.10](https://github.com/GoogleChromeLabs/ps-analysis-tool/releases/tag/v0.10.0)
